website/docs/d/privatelink_endpoint_service.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: private_endpoint_link"
+sidebar_current: "docs-mongodbatlas-datasource-private-endpoint-link"
+description: |-
+    Describes a Private Endpoint Link.
+---
+
+# mongodbatlas_privatelink_endpoint_service
+
+`mongodbatlas_privatelink_endpoint_service` describe a Private Endpoint Link. This represents a Private Endpoint Link Connection that wants to retrieve details in an Atlas project.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_privatelink_endpoint" "test" {
+  project_id    = "<PROJECT_ID>"
+  provider_name = "AWS"
+  region        = "us-east-1"
+}
+
+resource "aws_vpc_endpoint" "ptfe_service" {
+  vpc_id             = "vpc-7fc0a543"
+  service_name       = "${mongodbatlas_privatelink_endpoint.test.endpoint_service_name}"
+  vpc_endpoint_type  = "Interface"
+  subnet_ids         = ["subnet-de0406d2"]
+  security_group_ids = ["sg-3f238186"]
+}
+
+resource "mongodbatlas_privatelink_endpoint_service" "test" {
+  project_id            = "${mongodbatlas_privatelink_endpoint.test.project_id}"
+  private_link_id       = "${mongodbatlas_privatelink_endpoint.test.private_link_id}"
+  interface_endpoint_id = "${aws_vpc_endpoint.ptfe_service.id}"
+}
+
+data "mongodbatlas_privatelink_endpoint_service" "test" {
+  project_id            = "${mongodbatlas_privatelink_endpoint_service.test.project_id}"
+  private_link_id       = "${mongodbatlas_privatelink_endpoint_service.test.private_link_id}"
+  interface_endpoint_id = "${mongodbatlas_privatelink_endpoint_service.test.interface_endpoint_id}"
+}
+```
+
+## Argument Reference
+
+* `project_id` - (Required) Unique identifier for the project.
+* `private_link_id` - (Required) Unique identifier of the AWS PrivateLink connection.
+* `endpoint_service_id` - (Required) Unique identifier of the private endpoint service for which you want to create a private endpoint.
+* `provider_name` - (Required) Cloud provider for which you want to create a private endpoint. Atlas accepts `AWS` or `AZURE`.
+
+## Attributes Reference
```

Comment:
I think you have both AWS and Azure here but don't say which are for which, please show that.

Reply:
For `endpoint_service_id`  is the id generated from created `privatelink_endpoint`